storybook/stories/API/component/Tooltip.stories.tsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { ResponsiveContainer, Tooltip, LineChart, Line } from '../../../../src';
+import { pageData } from '../../data';
+import { getStoryArgsFromArgsTypesObject } from '../props/utils';
+
+type MyArgType = {
```

Comment:
nit: can we make this a common type (maybe `RechartsArgType`) that extends `ArgType` from storybook?

Reply:
Yep I'll do that in a followup PR.

Reply:
https://github.com/recharts/recharts/pull/3761